install/rpm/build.sh

```diff
@@ -26,25 +26,31 @@ stage_install_rpm() {
   prep_staging_rpm
   stage_install_common
   echo "Staging RPM install files in '${STAGEDIR}'..."
-  process_template "${BUILDDIR}/install/common/rpmrepo.cron" \
-    "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
-  chmod 755 "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
+  if [ "$CPANEL" = false ]; then
+    process_template "${BUILDDIR}/install/common/rpmrepo.cron" \
+      "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
+    chmod 755 "${STAGEDIR}/etc/cron.daily/${PACKAGE}"
+  fi
 
   # For CentOS, the load and conf files are combined into a single
   # 'conf' file. So we install the load template as the conf file, and
   # then concatenate the actual conf file.
   process_template "${BUILDDIR}/install/common/pagespeed.load.template" \
-    "${STAGEDIR}${APACHE_CONFDIR}/pagespeed.conf"
+    "${STAGEDIR}${APACHE_CONFDIR}/${CPANEL_CONF_PREFIX}pagespeed.conf"
```

Comment:
Shouldn't we rename CPANEL_CONF_PREFIX into something like PAGESPEED_CONF_PREFIX
It does not sit well with me to have very specifically named vars in the global flow.

Reply:
Agreed, considering there are other circumstances in which environments other than cPanel prefer a prefix.

Reply:
This has been addressed in the latest commit